Readme.md

```diff
@@ -75,6 +75,47 @@ install process be as simple as the `brew` process on macOS. See
 [#433](https://github.com/microsoft/scalar/issues/433) for progress on
 this feature._
 
+Installing on Linux
+-------------------
+
+Currently, we package a custom version of Git and Scalar as `.deb` packages
```

Comment:
Might want to call out that we've only built and tested for x86_64/amd64 architectures.